src/netspresso_trainer/models/base.py

```diff
@@ -67,6 +67,10 @@ def _get_name(self):
     def forward(self, x, label_size=None, targets=None):
         raise NotImplementedError
 
+    @abstractmethod
+    def deploy(self, ):
+        raise NotImplementedError
```

Comment:
I think better to implement deploy method in `TaskModel` class instead of subclasses.

Reply:
Yeah! Thank you for the review. I agree👍 It would prevent duplicates on the code base.

Reply:
I made a commit to resolve this conversation!